src/Interpreters/JIT/CHJIT.cpp

```diff
@@ -269,6 +269,14 @@ class AssemblyPrinter
     llvm::TargetMachine & target_machine;
 };
 
+static std::string DumpModuleIR(const llvm::Module& module)
```

Comment:
Wrap this in `PRINT_ASSEMBLY`?

Reply:
ok

Reply:
@rschu1ze Any hint? thanks.

Reply:
@rschu1ze I will add this pass manaually. So I close PR https://github.com/ClickHouse/llvm-project/pull/49.